### ISSUE TITLE
Test Mouse/other input Scenery changes

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/tasks#366]
I've put up http://www.colorado.edu/physics/phet/dev/olsonjb/friction_en.html and the iframed http://www.colorado.edu/physics/phet/dev/olsonjb/friction_en-iframe.html as testing for a potential fix for https://github.com/phetsims/scenery/issues/464. The change is basically us no longer telling the browser "Don't do anything else with the mouse/touch/etc. besides what we do" but instead "Please don't do X/Y/Z in addition to what we do".

Can both HTML files be tested for general input-related issues (mouse/touch/scroll wheel/stylus/etc.) across a range of platforms? I'm most interested in behavior that didn't happen before that now might happen, e.g.:
- Can you scroll/pan/zoom where it wasn't possible before (causing issues)?
- Can you select text/elements in the sim (like highlighting text)?
- Can you drag things that couldn't be dragged before?
- Does tapping highlight anything different than before?

Assigning to Ariel for prioritization/tagging (testing will be blocking for https://github.com/phetsims/scenery/issues/464).

**Comment by ariel-phet** [comment]
@elisemorgan can you hop on this testing this afternoon?

**Comment by elisemorgan** [comment]
@jonathanolson I can't seem to pull up the iframe link that you have here. The regular html link shows no issues, but as I remember from before this issue seems to be iframe specific

**Comment by jonathanolson** [comment]
Sorry, here's the URL: http://www.colorado.edu/physics/phet/dev/olsonjb/friction_en-iframe.html

**Comment by pixelzoom** [comment]
S2015R redeploys are on hold until this testing is completed.

**Comment by elisemorgan** [comment]
@jonathanolson the issue is now more difficult to recreate, but it is still present. I am not seeing any of the possible new behaviors that you listed above.

**Comment by jonathanolson** [comment]
> @jonathanolson the issue is now more difficult to recreate, but it is still present.

Can you provide instructions for recreating, including which browser version is used?

**Comment by elisemorgan** [comment]
iOS Yosemite  10.10.5 on the latest google chrome. Will test on more
platforms as soon as I get the chance. Drag around quickly, remove mouse
from frame, unclick. Like I said, it may take a few tries because it's not
as consistent as before.

**Comment by jonathanolson** [comment]
I haven't been able to reproduce with about ~15min of testing on my side (OS X 10.9.5, latest Chrome).

Can you describe the behavior that happened when you released the mouse button outside the frame? Did going back into the frame (with the mouse button up) still cause movement of the book itself?

**Comment by elisemorgan** [comment]
The book still moved when I had the mouse unclicked both inside and outside
the frame. I was able to move the mouse all over my screen and the book
moved. I was only able to get the issue to happen on the first click (ie I
kept refreshing the sim). Wish I could be more specific but I just did fast
movement in and out of the frame and then unclicked outside of the frame.

**Comment by elisemorgan** [comment]
@jonathanolson @ariel-phet I won't be available to do further testing of this until Monday. If testing needs to be completed before then, best assign to a different QA

**Comment by ariel-phet** [comment]
@elisemorgan, thanks for the heads up!

**Comment by jonathanolson** [comment]
I'm planning to work on reproducing tomorrow while on campus (with the device used if available, or otherwise just more up-to-date OS X laptops). Online searches into this behavior mostly came up with the preventDefault potential workaround, so I'm not sure what else to hit yet.

**Comment by ariel-phet** [comment]
@orejolaphet since you are here today can you work on this testing.

**Comment by ariel-phet** [comment]
@orejolaphet comment in this issue if you have any questions about the testing.

**Comment by orejolaphet** [comment]
Testing http://www.colorado.edu/physics/phet/dev/olsonjb/friction_en-iframe.html I was able to reproduce the issue on my machine (OSX 10.10.4 2015 macbook), but it only occurs when I use the "force touch" feature. The book only moved while my mouse was within the iframe even though unclicked.

**Comment by pixelzoom** [comment]
Since this is blocking RC testing, added "developer-meeting" label so that we remember to check on status.

**Comment by ariel-phet** [comment]
updated issue in #380, closing
